core/new-gui/src/app/workspace/types/visualization.interface.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * This file contains some type declaration for the VisualizationPanelContent component
+*/
+
+/**
+ * ChartType records all supported chart type in the frontend.
+ */
+export type ChartType = 'pie' | 'donut' | 'bar' | 'stacked bar' | 'word cloud'  | 'line' | 'spline';
```

Comment:
based on the other comment, this should be `export enum ChartType { PIE = 'pie', BAR = 'bar', .... }`. Then import this enum in `visualization-panel-content.component.ts`.